corehq/apps/users/models.py

```diff
@@ -382,6 +382,7 @@ class DomainMembership(Membership):
     override_global_tz = BooleanProperty(default=False)
     role_id = StringProperty()
     location_id = StringProperty()
+    assigned_location_ids = StringListProperty()
```

Comment:
Yes (?), as is the usual location_id. I took this as implied, should I ping Sheel for a spec?